drivers/hmis_external_apis/app/models/hmis_external_apis/external_forms/form_generator.rb

```diff
@@ -128,6 +131,12 @@ def render_choice_node(node)
       end
     end
 
+    def render_geolocation_node(node)
+      render_form_group(node: node) do
+        render_form_geolocation(label: node['text'], required: node['required'], name: node_name(node))
```

Comment:
The way I have it here, unlike the other `render` input methods, `render_form_geolocation` doesn't generate a new HTML id using `next_html_id` and pass it to the partial; instead, inside the partial, the html ID for the hidden input is generated and the label applies to that.

Maybe this is too topsy turvy, so I want to revisit it and match the existing pattern